### DISCUSSION TITLE
Bell Circuit Trouble Status

### DISCUSSION THREAD

**Discussion by BobRobertBob** [ufodone/envisalink_new#134] [Ideas]
I would like to request the addition of Bell Circuit Trouble status similar to AC Power and Battery.  The Bell circuit is monitored by the DSC panel (for an open circuit) and like the battery and AC power, it is useful to monitor it's status.

**Comment by ufodone** [comment]
I can look at adding it.  Would you mind opening an issue to track it?  Thanks!

**Comment by BobRobertBob** [comment]
Done.
